packages/fluxible-addons-react/package.json

```diff
@@ -27,7 +27,7 @@
   "peerDependencies": {
     "fluxible": ">=1.0.0",
     "react": "^16.3.0",
```

Comment:
What about `react` too?

Reply:
My fault, I was just about to sleep at that time :p